examples/step-2/step-2.cc

```diff
@@ -85,13 +85,13 @@ void make_grid (Triangulation<2> &triangulation)
                outer_radius = 1.0;
   GridGenerator::hyper_shell (triangulation,
                               center, inner_radius, outer_radius,
-                              10);
+                              5 );
```

Comment:
Fair enough. Can you mention in the comment above the `make_grid()` function that it is the same mesh as in step-1, but with fewer refinement steps?